pkg/core/native/native_test/cryptolib_verification_test.go

```diff
@@ -0,0 +1,924 @@
+	script[iIncrementEndOffset-1] = byte(iIterationStartOffset - iIncrementEndOffset + 2)
+	script[jIncrementEndOffset-1] = byte(jIterationStartOffset - jIncrementEndOffset + 2)
+	script[iIterationEndOffset-1] = byte(progRetOffset - iIterationEndOffset + 2)
+	script[jIterationEndOffset-1] = byte(iIncrementStartOffset - jIterationEndOffset + 2)
+
+	return script
+	// Here's an example of the resulting single witness invocation script (279 bytes length, the length may vary slightly depending on the number of m/n):
+	// NEO-GO-VM > loadbase64 VwcAQxMoBUkJQBPAcAwhAnDKJRC/powKR62KN6QDQp5tSIfoEjC8py27IcDuR1yaDCEDchjru3V6qYZQkyp0tlTEuajdXv7qERtzo96fQLR/zRwMIQOAXOe8VvBhB55uzHLBo3GWaLlzlHLLR0d4rsceFTuQGgwhAqWv1kjqoCHNDC5TNalaM1agtWH/vRDLLzsDq41K3B3aFMBxQcX7oOADAAAAAAEAAACeFI1BLVEIMBDOi3IQcxB0EHVtEy5VbHZuacouSQAYaG3OaW7OahTAHwwPdmVyaWZ5V2l0aEVDRHNhDBQb9XWrEYlohBNhCjWhKIbN4LZsckFifVtSJgprnHNunHQiB26cdiK2bZx1IqtJaxOz
+	// READY: loaded 279 instructions
+	// NEO-GO-VM 0 > ops
+	// INDEX    OPCODE       PARAMETER
+	// 0        INITSLOT     7 local, 0 arg    <<
+	// 3        DEPTH
+	// 4        PUSH3
+	// 5        JMPEQ        10 (5/05)
+	// 7        CLEAR
+	// 8        PUSHF
+	// 9        RET
+	// 10       PUSH3
+	// 11       PACK
+	// 12       STLOC0
+	// 13       PUSHDATA1    0270ca2510bfa68c0a47ad8a37a403429e6d4887e81230bca72dbb21c0ee475c9a
+	// 48       PUSHDATA1    037218ebbb757aa98650932a74b654c4b9a8dd5efeea111b73a3de9f40b47fcd1c
+	// 83       PUSHDATA1    03805ce7bc56f061079e6ecc72c1a3719668b9739472cb474778aec71e153b901a
+	// 118      PUSHDATA1    02a5afd648eaa021cd0c2e5335a95a3356a0b561ffbd10cb2f3b03ab8d4adc1dda
+	// 153      PUSH4
+	// 154      PACK
+	// 155      STLOC1
+	// 156      SYSCALL      System.Runtime.GetNetwork (c5fba0e0)
+	// 161      PUSHINT64    4294967296 (0000000001000000)
+	// 170      ADD
+	// 171      PUSH4
+	// 172      LEFT
+	// 173      SYSCALL      System.Runtime.GetScriptContainer (2d510830)
+	// 178      PUSH0
+	// 179      PICKITEM
+	// 180      CAT
+	// 181      STLOC2
+	// 182      PUSH0
+	// 183      STLOC3
+	// 184      PUSH0
+	// 185      STLOC4
+	// 186      PUSH0
+	// 187      STLOC5
+	// 188      LDLOC5
+	// 189      PUSH3
+	// 190      JMPGE        275 (85/55)
+	// 192      LDLOC4
+	// 193      STLOC6
+	// 194      LDLOC6
+	// 195      LDLOC1
+	// 196      SIZE
+	// 197      JMPGE        270 (73/49)
+	// 199      PUSHINT8     24 (18)
+	// 201      LDLOC0
+	// 202      LDLOC5
+	// 203      PICKITEM
+	// 204      LDLOC1
+	// 205      LDLOC6
+	// 206      PICKITEM
+	// 207      LDLOC2
+	// 208      PUSH4
+	// 209      PACK
+	// 210      PUSH15
+	// 211      PUSHDATA1    766572696679576974684543447361 ("verifyWithECDsa")
+	// 228      PUSHDATA1    1bf575ab1189688413610a35a12886cde0b66c72 ("NNToUmdQBe5n8o53BTzjTFAnSEcpouyy3B", "0x726cb6e0cd8628a1350a611384688911ab75f51b")
+	// 250      SYSCALL      System.Contract.Call (627d5b52)
+	// 255      JMPIFNOT     265 (10/0a)
```

Comment:
Then here you can
```
DUP
LDLOC3
ADD
STLOC3
```
False is 0, true is 1, you get exactly what you need, but this code would be executed irrespective of the outcome. Maybe we could even avoid some JMP.